datasets/wiki_asp/wiki_asp.py

```diff
@@ -0,0 +1,212 @@
+        ),
+        datasets.BuilderConfig(
+            name="artist", version=VERSION, description="A subset of dataset from the artist domain"
+        ),
+        datasets.BuilderConfig(
+            name="building", version=VERSION, description="A subset of dataset from the buildings domain"
+        ),
+        datasets.BuilderConfig(
+            name="company", version=VERSION, description="A subset of dataset from the company domain"
+        ),
+        datasets.BuilderConfig(
+            name="educational_institution",
+            version=VERSION,
+            description="A subset of dataset from the educational institution domain",
+        ),
+        datasets.BuilderConfig(
+            name="event", version=VERSION, description="A subset of dataset from the events domain"
+        ),
+        datasets.BuilderConfig(name="film", version=VERSION, description="A subset of dataset from the film domain"),
+        datasets.BuilderConfig(name="group", version=VERSION, description="A subset of dataset from the group domain"),
+        datasets.BuilderConfig(
+            name="historic_place", version=VERSION, description="A subset of dataset from the historic places domain"
+        ),
+        datasets.BuilderConfig(
+            name="infrastructure", version=VERSION, description="A subset of dataset from the infrastructure domain"
+        ),
+        datasets.BuilderConfig(
+            name="mean_of_transportation",
+            version=VERSION,
+            description="A subset of dataset from the transportation mean domain",
+        ),
+        datasets.BuilderConfig(
+            name="office_holder", version=VERSION, description="A subset of dataset from the office holder domain"
+        ),
+        datasets.BuilderConfig(name="plant", version=VERSION, description="A subset of dataset from the plant domain"),
+        datasets.BuilderConfig(
+            name="single", version=VERSION, description="A subset of dataset from the musical single domain"
+        ),
+        datasets.BuilderConfig(
+            name="soccer_player", version=VERSION, description="A subset of dataset from the soccer player domain"
+        ),
+        datasets.BuilderConfig(
+            name="software", version=VERSION, description="A subset of dataset from the software domain"
+        ),
+        datasets.BuilderConfig(
+            name="television_show", version=VERSION, description="A subset of dataset from the television show domain"
+        ),
+        datasets.BuilderConfig(name="town", version=VERSION, description="A subset of dataset from the town domain"),
+        datasets.BuilderConfig(
+            name="written_work", version=VERSION, description="A subset of dataset from the written work domain"
+        ),
+    ]
+
+    # Using Plant as default because of its small size (286MB)
+    # The full dataset is 28GB
+    DEFAULT_CONFIG_NAME = "plant"
+
```

Comment:
```suggestion
```
not sure we should have a default config here

Reply:
So we do not want to have a default for this dataset? This is multi-domain dataset where I chose the smallest domain as default. Happy to remove if you think we can do without default here.